core/soc/estimator.go

```diff
@@ -91,6 +93,15 @@ func (s *Estimator) RemainingChargeEnergy(targetSoC int) float64 {
 
 // SoC replaces the api.Vehicle.SoC interface to take charged energy into account
 func (s *Estimator) SoC(chargedEnergy float64) (float64, error) {
+	if charger, ok := s.charger.(api.Battery); ok {
+		f, err := charger.SoC()
+
+		if !errors.Is(err, api.ErrNotAvailable) {
```

Comment:
eigentlich if err == nil, oder? Falls es irgendeinen Fehler gäbe bliebe ja immer noch das Fallback zum Fahrzeug Soc?